backend/core-api/src/modules/logs/graphql/resolvers/queries.ts

```
@@ -1,22 +1,251 @@
+  }
+
+  if (operator === 'exists') {
+    return JSON.parse(value);
+  }
+
+  return value;
+};
 
 const generateFilters = (params) => {
-  let filter = {};
+  let filter: any = {};
```

ESLint

[error] 'filter' is never reassigned. Use 'const' instead.

(prefer-const)